stack/adding-a-new-package.rst

```diff
@@ -43,11 +51,33 @@ The roles assigned to these teams should typically be "Write" for "Data Manageme
   The automated builds use the team membership to determine the type of tag to be applied.
   Having the code reside in the ``lsst`` or ``lsst-dm`` organization on GitHub is not sufficient.
 
+repos.yaml
+----------
+
 The new package must be added to the `etc/repos.yaml file in the lsst/repos repository`_ along with its corresponding GitHub URL.
 This file is governed by a "self-merge" policy: upon opening a pull request, it will be checked by GitHub Actions, and developers may merge without further review on success.
-This change **must** be merged before the package can be built on Jenkins.
+This change **must** be merged before the package can be built on Jenkins, and this should be done prior to the RFC being filed.
```

Comment:
```suggestion
This change **must** be merged before the package can be built on Jenkins, and this should be done prior to the RFC being adopted.
```

Choosing the package name is part of the RFC process, so the name can't be used prior to filing the RFC.  Adoption is the earliest gate that could be used, but I could see an argument for not doing repos.yaml until final implementation.

Reply:
I changed it to "early in the implementation process", since it's a prereq for a lot of other things.